documents-d-archeologie-francaise.csl

```diff
@@ -357,8 +357,8 @@
   </macro>
   <macro name="collection">
     <group delimiter=", ">
-      <text variable="collection-title"/>
-      <text variable="collection-number"/>
+<text variable="collection-title" prefix=", coll. « " suffix=" »"/>
```

Comment:
it would be much better to set the guillemets using ```quotes="true"```, i.e. in this case
```<text variable="collection-title" prefix=", coll. " quotes="true"/>``` among other advantages, it makes the space between guillemet and title non-breaking, so that you won't get a line break right there. 

Also, if you could indent this properly, that'd be great. You can use: http://rintze.zelle.me/style-formatter/ to do this for all lines with one click.